components/product/ProductCard/index.tsx

```diff
@@ -77,15 +77,13 @@ function ProductCard({
   const id = `product-card-${productID}`;
 
   const { hasDiscount, hasMultiplePrices } = useMemo(() => {
-    const variantPrices = product.isVariantOf?.hasVariant.map(
-      (item) => item.offers?.offers?.[0]?.price,
-    );
+    const variantPrices = {
+      minPrice: product.offers?.offers[0].price,
+      maxPrice: product.offers?.offers[0].priceSpecification[0].price,
```

Comment:
Isso aqui pode ser variavel, a forma correta é procurar pelo priceType que voce quer, isso pode causar side effects no futuro
```suggestion
      maxPrice: product.offers?.offers[0].priceSpecification[0].price,
```